src/client/Microsoft.Identity.Client/Internal/Requests/ClientCredentialRequest.cs

```diff
@@ -189,16 +204,51 @@ private async Task<AuthenticationResult> SendTokenRequestToAppTokenProviderAsync
 
         private async Task<MsalAccessTokenCacheItem> GetCachedAccessTokenAsync()
```

Comment:
nit: can you add `/// <summary>` docs for this function?

Reply:
If you choose to implement this feedback, can you also do it for the other new functions you've created in this file?